PluralKit.Bot/Commands/Groups.cs

```diff
@@ -217,7 +219,8 @@ await ctx.Reply(embed: new EmbedBuilder()
                         $"To print the description with formatting, type `pk;group {target.Reference(ctx)} description -raw`."
                         + (ctx.System?.Id == target.System
                             ? $" To clear it, type `pk;group {target.Reference(ctx)} description -clear`."
-                            : "")))
+                            : "")
+                            + $" Using {target.Description.Length}/{Limits.MaxDescriptionLength} characters."))
```

Comment:
this should probably be own-system only

Reply:
Gotcha, I applied the same pattern to members/systems too so I’ll change that as well